### DISCUSSION TITLE
Add TTL support per client

### DISCUSSION THREAD

**Discussion by kotyara85** [ory/hydra#3101] [Ideas]
Describe your problem
Currently TTL for tokens is specified globally, which is not convenient as sometimes you want to have different TTL per provider (client) 

Describe your ideal solution
Add TTL params to hydra clients create command

clients create \
  --endpoint http://hydra:4445 --id client-id --secret secret \
  --callbacks https://callback.url \
  --ttl-login-consent-request=5h
  --ttl-access-token=5h
  --ttl-id-token=5h
  --ttl-refresh-token=5h

Workarounds or alternatives
We have to deploy a separate instance for each ttl

Version
v1.11.7

**Comment by vinckr** [comment]
I think that would be a good feature and we have some plans to implement this. 
See also this issue for an older discussion: https://github.com/ory/hydra/issues/1529
